src/packages/documents/documents/workspace/views/info/document-workspace-view-info-history.element.ts

```diff
@@ -88,6 +95,16 @@ export class UmbDocumentWorkspaceViewInfoHistoryElement extends UmbLitElement {
 		this.#getLogs();
 	}
 
+	#onRollbackModalOpen = () => {
+		const modalContext = this.#modalContext?.open(this, UMB_ROLLBACK_MODAL, {});
```

Comment:
Consider calling on the modal manager with `this.getContext(UMB_MODAL_MANAGER_CONTEXT)` rather than having to listen on it constantly, since this is a one-off method that is only executed when you click on something else.